src/views/InteractiveMap/Legend.vue

```diff
@@ -51,12 +51,18 @@
             Not in the model
           </p>
         </v-layout>
-        <v-layout align-center v-show="!showScoreLegend">
+        <v-layout align-center v-show="!showScoreLegend && !showEnzymeUsage">
           <div class="measured-flux"></div>
           <p class="mb-0 ml-3">
             Measured flux
           </p>
         </v-layout>
+        <v-layout align-center v-show="showEnzymeUsage">
+          <div class="measured-flux"></div>
+          <p class="mb-0 ml-3">
+            Enzyme usage exceeds threshold
```

Comment:
```
Reaction with enzyme usage greater than or equal to threshold
```
Hopefully the legend still looks decent with the longer description?

Reply:
![image](https://user-images.githubusercontent.com/64745/70632689-9958dd80-1c2f-11ea-95ee-648a17b6294a.png)

WDYT? It looks a bit long to me but maybe I'm just too used to the previous size. At least this makes it easier to distinguish the different gradient colors.

Reply:
LGTM!